### PR TITLE
#5768 [Manager] Supply a default prefix value for runTest script

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #issue_number)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/utils/commonUtils.js [hunk 1]
```diff
@@ -21,3 +22,13 @@ exports.sleep = (ms) => {
 exports.randomPort = ({ min = 5000, max = 65535 } = {}) => {
   return Math.floor(Math.random() * (max - min + 1)) + min;
 };
+
+exports.logger = {
```
Comment (konekoya) [inline ohara-manager/utils/commonUtils.js:26]: We're doing a lot of logging in our scripts, let's wrap this into a small utility so we don't have to worry about the linter complains

File: ohara-manager/scripts/runTests.js [hunk 2]
```diff
@@ -167,66 +171,57 @@ const run = async (options) => {
...
-    // Ignore the error message, and exit with a fail status
-    process.exit(1);
-  }
 };
 
 const testMode = process.env.CYPRESS_TEST_MODE;
-const defaultEnv = utils.getDefaultEnv();
-const prefix = servicePrefix ? servicePrefix : defaultEnv.servicePrefix;
+const defaultPrefix = getDefaultEnv().servicePrefix;
+const prefix = servicePrefix ? servicePrefix : defaultPrefix || 'local';
```
Comment (konekoya) [inline ohara-manager/scripts/runTests.js:212]: Supply a default value for prefix

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-it

**Review by eechih — APPROVED** [review]

**Comment by konekoya** [comment]
retry manager-it

**Pull request merged**